consolelabs/mochi-api
# chore: improve CreateGuildTokenRole handler logic

**What does this PR do?**

-   [x] refactor package name from guild_config_token_role
-  [x] Improve  logic  in CreateGuildTokenRole / RemoveGuildTokenRole handler by adding the capability to update a role using `/tokenrole set` `/tokenrole remove`, and automatically assign roles to Discord users in the background after creating , updating, or deleting a role.

**How to test**

-   use "/tokenrole set" or "/tokenrole remove" to check whether or not an existed roles updated, and users's roles should be updated in discord afterward
